vunit/ui.py

```diff
@@ -548,6 +578,31 @@ def set_pli(self, value):
     def add_source_files(self, pattern, preprocessors=None):
         self._parent.add_source_files(pattern, self._library_name, preprocessors)
 
+    def generate_codecs(self, package_name, codec_package_name=None, used_packages=None, output_file_name=None):
```

Comment:
I have already fixed it for you in the `com-fixes` branch. Use that and save some work.